src/site/apt/development_contribution.apt

```diff
@@ -114,9 +132,25 @@ Contributing Code
   they are pushing on to the repository.
   Often the committer will also be the reviewer for non-committer changes.
 
+    * Add ASF as a git remote (first time committers).
+
+      Create a git remote for ASF repository. {{{https://git-wip-us.apache.org/repos/asf/vxquery.git}}}
+
     * Double check the VXQuery and XQTS tests.
 
+      A {{{https://asterix-jenkins.ics.uci.edu/job/vxquery-pr/}VXQuery Jenkins}} instance has been set up to
+      check the last ten Pull Requests.
```

Comment:
What are we checking there? Should we add that to the docs or does it get outdated too fast?